docs/infrastructure_and_maintenance/clustering/ibexa_cloud_and_ddev.md

```diff
@@ -0,0 +1,94 @@
+
+## Simulate Ibexa Cloud using the `ddev-platformsh` add-on
+
+To configure the [`ddev/ddev-platformsh` add-on](https://github.com/ddev/ddev-platformsh), you'll need a [Platform.sh API Token](https://docs.platform.sh/administration/cli/api-tokens.html).
+
+`COMPOSER_AUTH` from PLatform.sh can't be properly used as JSON commas will be badly interpreted by `--web-environment-add` which would see them as multiple variables' separators. But, it must exist for Platform.sh hooks' scripts to work. An auth.json file can be used, see [Using an auth.json](../../getting_started/install_using_ddev.md#using-an-authjson) for more.
+
+This example sequence will
+
+- download the Ibexa Cloud Platform.sh project from the default environment "production" into a new directory (replace `<project-ID>` with the hash of your own project, see [`platform help get`](https://docs.platform.sh/administration/cli.html#3-use) for options like selecting another environment),
+- config a new DDEV project,
+- ignore `.ddev/` directory from Git,
+- set Composer authentication using an already existing `auth.json` file,
+- create a `public/var` directory if it doesn't exist (to allow the creation of `public/var/.platform.installed` by Platform.sh hook script),
+- install the `ddev/ddev-platformsh` add-on which will prompt for the Platform.sh API token, project ID and environment name,
+- comment the NodeJS and NVM installations from the hooks copied in `.ddev/config.platformsh.yaml`,
+- change `maxmemory-policy` from default `allkeys-lfu` to a [value accepted by the `RedisTagAwareAdapter`](https://github.com/symfony/cache/blob/5.4/Adapter/RedisTagAwareAdapter.php#L95),
+- start the project
+- get the content from Platform.sh, both database and binary files using `ddev pull platform` feature from the add-on,
+- restart the project,
+- display information about the project services,
+- open the project into a browser.
+
+```bash
+platform project:get <project-ID> my-ddev-project && cd my-ddev-project
+ddev config --project-type=php --php-version 8.1 \
+  --docroot=public --create-docroot \
+  --http-port=8080 --https-port=8443 \
+  --web-environment-add DATABASE_URL=mysql://db:db@db:3306/db \
+  --web-environment-add CACHE_POOL=cache.redis \
+  --web-environment-add CACHE_DSN=redis \
+  --web-environment-add COMPOSER_AUTH=''
+echo '.ddev/' >> .gitignore
+cp <path-to-an>/auth.json .ddev/homeadditions/
+if [ ! -d public/var ]; then mkdir public/var; fi
+ddev get ddev/ddev-platformsh
+sed -i -E "s/( +)(.*nvm (install|use).*)/\1#\2/" .ddev/config.platformsh.yaml
+sed -i 's/maxmemory-policy allkeys-lfu/maxmemory-policy volatile-lfu/' .ddev/redis/redis.conf
+ddev start
+ddev pull platform -y
+ddev restart
+ddev describe
+ddev launch
+```
+
+!!! note
+
+    As the Platform.sh API token is set at user profile level, it will be stored globaly under current user root as `PLATFORMSH_CLI_TOKEN` in ~/.ddev/global_config.yaml.
```

Comment:
```suggestion
    As the Platform.sh API token is set at user profile level, it will be stored globally under current user root as `PLATFORMSH_CLI_TOKEN` in `~/.ddev/global_config.yaml`.
```